gradescope_utils/autograder_utils/gradescope_django_runner.py

```diff
@@ -1,13 +1,15 @@
-import sys
-from django.test.runner import DiscoverRunner
 from gradescope_utils.autograder_utils.json_test_runner import JSONTestRunner
+from django.test.runner import DiscoverRunner
+from django.conf import settings
 
 
 class GradescopeDjangoRunner(DiscoverRunner):
     """Replacing Django Default Unit Test Runner with Gradescope Test Runner"""
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        kwargs = getattr(settings, "GRADESCOPE_PARAMETERS", {})
+        self.test_runner = JSONTestRunner(**kwargs)
```

Comment:
I know it's not being used currently, but I would still find it cleaner to not overwrite `kwargs` since that could be confusing.

```suggestion
        parameters = getattr(settings, "GRADESCOPE_PARAMETERS", {})
        self.test_runner = JSONTestRunner(**parameters)
```